test/mechanics/mesh/UnitMeshFem.cpp

```diff
@@ -28,6 +28,36 @@ void Check2DMesh(NuTo::MeshFem& mesh)
     BOOST_CHECK_NO_THROW(transformedMesh.NodeAtCoordinate(Eigen::Vector2d(43., 9.)));
 }
 
+BOOST_AUTO_TEST_CASE(MeshTrusses)
+{
+    auto mesh = NuTo::UnitMeshFem::CreateTrusses(13);
+    BOOST_CHECK_EQUAL(mesh.Elements.Size(), 13);
+    BOOST_CHECK_EQUAL(mesh.Nodes.Size(), 14);
+
+    bool nodesHaveExpectedValue = true;
+    for (unsigned int i = 0; i < mesh.Nodes.Size(); ++i)
+    {
+        if (std::abs(mesh.Nodes[i].GetValues()[0] - static_cast<double>(i) / mesh.Elements.Size()) > 1e-6)
+        {
+            nodesHaveExpectedValue = false;
+            break;
+        }
+    }
+    BOOST_CHECK(nodesHaveExpectedValue);
+
+    bool correctNodeOrder = true;
+    for (auto& element : mesh.Elements)
```

Comment:
I like that you check the elements as well. I forgot that in the other tests in this file. But what is the actual requirement? I'd say that it is not a specific ordering, but a positive determinant of the jacobian.

Reply:
If you are checking 2d or 3d elements, checking J is probably smarter. However, for a linear 1d element there are only 2 ways to store the nodes, one that gives you a negative J and one that gives you a positive J. Calculating J requires an interpolation, which we don't have at this point, right?